### PR TITLE
test: add unit tests for azureconst.go

### PR DESCRIPTION
**Pull request by tariq1890**
Add unit tests to increase overall code coverage

### CODE REVIEW

**Comment by mboersma** [comment]
This looks like it just adds tests&mdash;which is fantastic&mdash;rather than refactoring, so shouldn't the commit type be `test:`?

**Comment by tariq1890** [comment]
@mboersma you're right. WIll correct it.

**Comment by tariq1890** [comment]
@mboersma done! Requesting your LGTM :)

File: pkg/helpers/azureconst_test.go [hunk 1]
```diff
@@ -4,10 +4,289 @@
 package helpers
 
 import (
+	"encoding/json"
 	"strings"
 	"testing"
 )
 
+func TestGetAzureLocations(t *testing.T) {
+	expectedLocationMap := map[string]struct{}{
```
Comment (mboersma) [inline pkg/helpers/azureconst_test.go]: It's annoying to duplicate these big data structures, but I guess that's correct black box testing.

**Comment by jackfrancis** [comment]
/lgtm

pending unit tests

see what I did there?

**Pull request merged**